src/CurrencyNetwork.ts

```diff
@@ -31,6 +40,7 @@ export class CurrencyNetwork {
 
   private provider: TLProvider
 
+  /** @hidden */
```

Comment:
Just for me to get this: when do you hide something manually that should not be private in the code? :thinking:

Reply:
So currently some methods need to be publicly available for internal use within the library source code, such as the constructors for the subclasses, but should not directly be used externally by the user... Can be improved in the future when we refactor the clientlib